BibBuddy/app/src/main/java/de/bibbuddy/BookFragment.java

```diff
@@ -49,6 +49,7 @@
   private ExportBibTex exportBibTex;
   private final String folderName = "Download";
   private final String fileTypeBibToast = ".bib";
+
   private static final int STORAGE_PERMISSION_CODE = 1;
```

Comment:
The folderName, fileTypeBibToast and STORAGE_PERMISSION_CODE always have the same values. Maybe you can create a ExportConstants class for them (e.g. see LibraryKeys class) and also use them in your other export code. Then you could use your constants like this: ExportConstants.STORAGE_PERMISSION_CODE